pipes/WDL/workflows/tasks/tasks_assembly.wdl

```diff
@@ -184,7 +184,7 @@ task refine {
         # prep GATK
         mkdir gatk
         if [[ ${gatk_jar} == *.tar.bz2 ]]; then
-          tar -xjvf ${gatk_jar} -C gatk
+          tar -xjvf ${gatk_jar} -C gatk --no-same-owner
```

Comment:
@tomkinsc   not sure matters, but 'tar -o' is more portable: https://github.com/npm/npm/issues/1128

Reply:
Thanks! Thought I checked this but apparently I have GNU tar installed on my Mac...